src/components/Button.js

```diff
@@ -1,18 +1,22 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useContext } from 'react';
 import { StyleSheet } from 'react-native';
 import { Button as RNEButton } from 'react-native-elements';
 
 import { colors, normalize } from '../config';
 import { DiagonalGradient } from './DiagonalGradient';
+import { OrientationContext } from '../OrientationProvider';
 
 export const Button = ({ title, onPress }) => {
+  const { orientation } = useContext(OrientationContext);
```

Comment:
empty line before return, please
i will update it

Reply:
thanks